sdk/appconfiguration/azure-spring-cloud-appconfiguration-config/src/main/java/com/azure/spring/cloud/config/feature/management/entity/Feature.java

```diff
@@ -19,16 +21,16 @@
     private String key;
 
     @JsonAlias("enabled-for")
-    private HashMap<Integer, FeatureFilterEvaluationContext> enabledFor;
+    private HashMap<Integer, FeatureFlagFilter> enabledFor;
```

Comment:
Will this be a API breaking change for the users? Will this pojo be used directly in users' codes?

Reply:
No, I tested it out and it works fine. We convert it to a map before placing it as a property.